gemfiles/activerecord_6.1.gemfile.lock

```diff
@@ -2,12 +2,26 @@ PATH
   remote: ..
   specs:
     authie (0.0.0.dev)
+      actionpack (>= 5.0, < 8.0)
```

Comment:
I believe all the `activerecord*.` files here should be removed, as the Appraisal groups have been renamed to `rails*`.